src/java/map/BinaryMask.java

```diff
@@ -65,6 +67,7 @@ public BinaryMask randomize(float density) {
 			}
 		}
 		applySymmetry();
+		VisualDebugger.visualizeMask(this);
```

Comment:
what if instead of duplicating this a lot of times we do it in the concurrent mask that is calling this?

Reply:
Then it wouldn't work with non-concurrent masks right?

Reply:
Hmm, right.

Reply:
That would still work for the "add visualization as a method to masks" approach though.

Reply:
What do you mean?